ansible/roles/ocp4-workload-debezium-demo/tasks/provision_e2e_demo.yaml

```diff
@@ -76,11 +82,11 @@
     api_version: kafka.strimzi.io/v1beta1
   register: kafka_resource
   until: >-
-    (kafka_resource.resources is defined) 
+    (kafka_resource.resources is defined)
```

Comment:
Can this just be turned into a list which will be a logical and? This is okay, but I think the list is more readable.

Reply:
Oh, I think I can do much better than that... I was trying to make minimal changes, but now that you mention...